### PR TITLE
add a Windows warning

### PR DESCRIPTION
**Pull request by karlitschek**
Add a "soft" Windows warning.

### CODE REVIEW

**Comment by karlitschek** [comment]
@DeepDiver1975 @schiesbn @PVince81

File: settings/admin.php [hunk 1]
```diff
@@ -81,6 +81,10 @@
 $databaseOverload = (strpos(\OCP\Config::getSystemValue('dbtype'), 'sqlite') !== false);
 $template->assign('databaseOverload', $databaseOverload);
 
+// warn if Windows is used
+$WindowsWarning = strtoupper(substr(PHP_OS, 0, 3)) === 'WIN';
```
Comment (DeepDiver1975) [inline settings/admin.php]: please use
```php
OC_Util::runningOnWindows()

Comment (nickvergessen) [inline settings/admin.php]: fixed

File: settings/templates/admin.php [hunk 2]
```diff
@@ -125,6 +125,20 @@
 <?php
 }
 
+// Windows Warning
+if ($_['WindowsWarning']) {
+	?>
+<div class="section">
+	<h2><?php p($l->t('Windows'));?></h2>
```
Comment (nickvergessen) [inline settings/templates/admin.php]: Should the caption be a little more detailed?

**Comment by nickvergessen** [comment]
Kind of unfortunate to add translated strings so late into the game?

File: settings/templates/admin.php (hunk 2 above)
Comment (DeepDiver1975) [inline settings/templates/admin.php]: suggestions?

**Comment by DeepDiver1975** [comment]
> Kind of unfortunate to add translated strings so late into the game?

true - but low prio on admin section and on windows :speak_no_evil:

File: settings/templates/admin.php (hunk 2 above)
Comment (LukasReschke) [inline settings/templates/admin.php]: I already see our translators translating this to "Fenster" – we also have a "Datenkbankhochrüstung" and a "Aktualisierer" ;-)

We also can somehow add hints for translators, see http://doc.owncloud.org/server/7.0/developer_manual/app/l10n.html?highlight=translator#hints

```php
<ul id="translations">
    <li id="add-new">
        <?php
            // TRANSLATORS Will be shown inside a popup and asks the user to add a new file
            p($l->t('Add new file'));
        ?>
    </li>
</ul>
```

Comment (nickvergessen) [inline settings/templates/admin.php]: > ### Known issues while running ownCloud on windows
> Your server is running on Windows. We highly recommend Linux for better user experience.

File: settings/admin.php [hunk 3]
```diff
@@ -81,6 +81,10 @@
 $databaseOverload = (strpos(\OCP\Config::getSystemValue('dbtype'), 'sqlite') !== false);
 $template->assign('databaseOverload', $databaseOverload);
 
+// warn if Windows is used
+$runningOnWindows = OC_Util::runningOnWindows();
+$template->assign('WindowsWarning', $runningOnWindows);
+$template->assign('WindowsWarning', true);
```
Comment (nickvergessen) [inline settings/admin.php]: what happened here now?

Comment (DeepDiver1975) [inline settings/admin.php]: bäääh - test code

Comment (DeepDiver1975) [inline settings/admin.php]: thx for noticing

**Comment by DeepDiver1975** [comment]
@nickvergessen @LukasReschke please review once more - THX

**Comment by LukasReschke** [comment]
:+1:

**Comment by MorrisJobke** [comment]
:+1:

File: settings/admin.php (hunk 1 above)
Comment (karlitschek) [inline settings/admin.php]: Nice. I missed that we have this call already :-)

**Pull request merged**